src/pages/main/components/carousel/CarouselImg.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const CarouselImg = ({ imgData, curImg, fadeIn }) => {
+  return (
+    <div className={`carouselImg ${fadeIn === true ? 'active' : 'hidden'}`}>
```

Comment:
```suggestion
    <div className={`carouselImg ${fadeIn ? 'active' : 'hidden'}`}>
```
rue일 때 true를, false일 때 false를 리턴하는 경우 굳이 써주지 않아도 될 것 같습니다.